docs/miscellaneous/create-dns-stamp.md

```diff
@@ -0,0 +1,79 @@
+# How to Create Your Own DNS Stamp for Secure DNS
+
+This guide will help you learn how to create your own DNS stamp for Secure DNS. Secure DNS is a service that enhances your internet security and privacy by encrypting your DNS queries. This prevents the possibility of your queries being intercepted or manipulated by malicious actors.
+
+For Secure DNS, `tls://`, `https://` or `quic://` URLs are normally used and it is enough for most users, and is recommended way.
+
+However, if you need additional security, like pre-resolved server IPs and certificate pinning by hash, you may generate your own DNS stamp. This guide tells you how to do it.
+
+## Introduction to DNS Stamps
+
+DNS Stamps are short strings that include all the information needed to connect to a secure DNS server. They simplify the process of setting up Secure DNS as the user does not need to manually enter all this data. 
+
+DNS stamps allow you to customize secure DNS settings beyond the usual URLs. In particular, they allow you to specify hard-coded server addresses, use certificate hashing, and so on. These features make DNS stamps a more robust and versatile option for configuring secure DNS settings.
+
+## Choosing the Protocol
+
+Types of secure DNS include `DNS-over-HTTPS (DoH)`, `DNS-over-QUIC (DoQ)`, and `DNS-over-TLS (DoT)` and some others. Choosing one of these protocols depends on your preferences and usage conditions.
+
+## Creating a DNS Stamp
+
+1. Open the [DNSCrypt Stamp Calculator](https://dnscrypt.info/stamps/).
+
+2. Depending on the chosen protocol, select the corresponding protocol from the dropdown menu (DoH, DoT, or DoQ).
+
+3. Fill in the necessary fields:
+    - **IP address**: 
+    **Note**: This field is optional and should be used with caution: using this option can break the Internet on IPv6-only networks.
```

Comment:
now we use admonitions to add notes to documentation: http://docusaurus.io/docs/markdown-features/admonitions

Reply:
can break —> may disrupt